### PR TITLE
Add Ceph roadmap items

### PR DESCRIPTION
**Pull request by cholcombe973**
@javacruft Let me know what you think of these.

### CODE REVIEW

**Comment by javacruft** [comment]
Reviewing now; please could you move them into the 'newton' specs folder as the intent is to have specs release aligned, rather than function aligned.

File: specs/ceph/approved/ceph-broker.rst [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+  any of the sections in this template.  If you have nothing to say
+  for a whole section, just write: "None". For help with syntax, see
+  http://sphinx-doc.org/rest.html To test out your formatting, see
+  http://www.tele3.cz/jbar/rest/rest.html
+
+===============================
+Ceph Broker
+===============================
+
+Connect Ceph to existing Openstack’s
```
Comment (javacruft) [inline specs/ceph/approved/ceph-broker.rst]: I thought this was the other way around? Connect a Juju deployed OpenStack to an existing Ceph?

File: specs/ceph/approved/ceph-broker.rst [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+Some customers have asked how to connect a Juju deployed Ceph to an existing
+OpenStack cluster that was deployed with home grown or someone else’s tooling.
+This causes a problem in that it is not possible to relate Ceph to the
+OpenStack cluster.
+
+Proposed Change
+===============
+
+The idea here is to create a charm that can provide the information that Openstack
+needs to connect to the existing Ceph cluster.
```
Comment (javacruft) [inline specs/ceph/approved/ceph-broker.rst]: Idea sounds a bit vague :-) maybe use some more directional word:

 'A new charm will be created that acts a a broker between an existing Ceph deployment, and a Juju deployed OpenStack Cloud;  The charm will provide the same relations as the existing ceph-mon charm.'

File: specs/ceph/approved/ceph-broker.rst [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+
+Alternatives
+------------
+
+The alternative is manually connecting the Ceph and OpenStack together.  This is
+fine for some customers but not acceptable for bootstack.
```
Comment (javacruft) [inline specs/ceph/approved/ceph-broker.rst]: Its also not particularly manageable is the pertinent point here I think.

File: specs/ceph/approved/ceph-broker.rst [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+.. code-block:: bash
+
+    git-review -t ceph-broker
+
+Work Items
+----------
+
+Work items or tasks -- break the feature up into the things that need to be
+done to implement it. Those parts might end up being done by different people,
+but we're mostly trying to understand the timeline for implementation.
```
Comment (javacruft) [inline specs/ceph/approved/ceph-broker.rst]: Can we get at least a high level breakdown here; I'd like to see some sort of automated testing still if possible.

File: specs/ceph/approved/ceph-broker.rst [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+------------
+
+Will any new git repositories need to be created?
+
+Documentation
+-------------
+
+Will this require a documentation change?  If so, which documents?
+Will it impact developer workflow?  Will additional communication need
+to be made?
```
Comment (javacruft) [inline specs/ceph/approved/ceph-broker.rst]: Boilerplate code...

File: specs/ceph/approved/ceph-broker.rst [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+----------
+
+Work items or tasks -- break the feature up into the things that need to be
+done to implement it. Those parts might end up being done by different people,
+but we're mostly trying to understand the timeline for implementation.
+
+Repositories
+------------
+
+Will any new git repositories need to be created?
```
Comment (javacruft) [inline specs/ceph/approved/ceph-broker.rst]: Boilerplate code...

File: specs/ceph/approved/ceph-broker.rst [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+
+Will this require a documentation change?  If so, which documents?
+Will it impact developer workflow?  Will additional communication need
+to be made?
+
+Security
+--------
+
+Does this introduce any additional security risks, or are there
+security-related considerations which should be discussed?
```
Comment (javacruft) [inline specs/ceph/approved/ceph-broker.rst]: Boilerplate code...

File: specs/ceph/approved/ceph-broker.rst [hunk 8]
```diff
@@ -0,0 +1,94 @@
...
+
+Testing
+-------
+
+What tests will be available or need to be constructed in order to
+validate this?  Unit/functional tests, development
+environments/servers, etc.
```
Comment (javacruft) [inline specs/ceph/approved/ceph-broker.rst]: Boilerplate code...

File: specs/ceph/approved/ceph-broker.rst (hunk 1 above)
Comment (cholcombe973) [inline specs/ceph/approved/ceph-broker.rst]: Yeah I keep getting this mixed up in my head :)

**Pull request merged**